google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ServerToIDEFileResolver.java

```diff
@@ -75,11 +76,15 @@ public VirtualFile getFileFromPath(@NotNull Project project, @NotNull String pat
     VirtualFile file = FILE_SYSTEM.findFileByPath(project.getBasePath() + "/" + path);
     // Try class name with package and class file name.
     if (file == null) {
-      PsiClass[] matchingClasses = JavaPsiFacade.getInstance(project)
-          .findPackage(getPackageFromPath(path))
-          .findClassByShortName(getClassNameFromPath(path), GlobalSearchScope.allScope(project));
-      if (matchingClasses.length > 0) {
-        file = matchingClasses[0].getContainingFile().getVirtualFile();
+      PsiPackage psiPackage = JavaPsiFacade.getInstance(project)
```

Comment:
if it is null should we log something? or notify the user that the class can't be found?